src/dom.js

```diff
@@ -270,3 +271,24 @@ export function childElementByTag(parent, tagName) {
     return el.tagName == tagName;
   });
 }
+
+
+/**
+ * Returns element attributes that pass the callback test as params.
+ * @param {!Element} element
+ * @param {function(!Element):string|null} parseAttributeCallback
+ * @return {Object}
+ */
+export function getAttributesAsParams(element, parseAttributeCallback) {
```

Comment:
In that case, do we really need a filter or can we just automatically test for data-param- prefix?